README.md

```diff
@@ -4,8 +4,23 @@ skelcd-control-leanos
 [![Travis Build](https://travis-ci.org/yast/skelcd-control-leanos.svg?branch=master)](https://travis-ci.org/yast/skelcd-control-leanos)
 
 
-Installation control file for SUSE Linux Enterprise Server product
+Installation control file for SUSE Linux Enterprise LeanOS product
 
 See also the [documentation for the `control.xml` file][1].
 
 [1]: https://github.com/yast/yast-installation/blob/master/doc/control-file.md
+
+## How It Works
+
+LeanOS integrate multiple previously independ products. LeanOS have its control.xml file
+which have reasonable defaults, except partitioning. And its workflow ends up with product selection client.
+And here comes into play product specific installation.xml. It is common add-on installation xml, that can do
+everythin any add-on can do. Also LeanOS can be installed itself, so it have its control.xml and
```

Comment:
What about something like:

LeanOS adds the capability to install independent products with the same media.

It has its own control.xml file which has reasonable defaults for almost all except partitioning.

Its workflow ends up with a product selection client, and there is where comes to play the `installation.xml` specific product file. It is a common add-on installation xml that can do everything an add-on can do. You can read more about the add-on installation.xml in this [document](https://github.com/yast/yast-installation/blob/master/doc/control-file.md#control-sectionadd-on-product-installation-workflow-specification)

But LeanOS is a product itself so apart of the `control.xml` file it also has its own `installation.xml`.